pkg/bootstrap/bootstrap_test.go

```diff
@@ -39,7 +40,36 @@ var _ = Describe("Bootstrap", func() {
 			output, err := cmd.CombinedOutput()
 			Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("init failed: %s", string(output)))
 
-			Expect(bootstrap.CreateConfig("foo", "bar", temp)).To(Succeed())
+			Expect(bootstrap.CreateConfig(bootstrap.Config{
+				GitRepository: profilesv1.GitRepository{
+					Name:      "bar",
+					Namespace: "foo",
+				},
+			}, temp)).To(Succeed())
+			data, err := ioutil.ReadFile(filepath.Join(temp, ".pctl", "config.yaml"))
+			Expect(err).ToNot(HaveOccurred())
+
+			var config bootstrap.Config
+			Expect(yaml.Unmarshal(data, &config)).To(Succeed())
+			Expect(config).To(Equal(bootstrap.Config{
+				GitRepository: profilesv1.GitRepository{
+					Name:      "bar",
+					Namespace: "foo",
+				},
+			}))
+		})
+		It("creates the config file with default location", func() {
+			cmd := exec.Command("git", "init", temp)
+			output, err := cmd.CombinedOutput()
+			Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("init failed: %s", string(output)))
+
+			Expect(bootstrap.CreateConfig(bootstrap.Config{
+				GitRepository: profilesv1.GitRepository{
+					Name:      "bar",
+					Namespace: "foo",
+				},
+				DefaultDir: "default-dir",
+			}, temp)).To(Succeed())
```

Comment:
does this need to be a separate test? I feel like its an extension of the above test

Reply:
It doesn't have to, no.